src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -21,6 +21,11 @@ internal class ManagedProjectSystemPackage : AsyncPackage
                                                   ProjectCapability.HandlesOwnReload + "; " +
                                                   ProjectCapability.OpenProjectFile;
 
+        public const long RunCodeAnalysisTopLevelBuildMenuCmdId = 0x066f;
+        public const long RunCodeAnalysisProjectContextMenuCmdId = 0x0670;
+        public const string CodeAnalysisPackageGuid = "B20604B0-72BC-4953-BB92-95BF26D30CFA";
+        public const string VSStd2KCommandSet = "1496A755-94DE-11D0-8C3F-00C04FC2AAE2";
```

Comment:
This should already exist: https://github.com/dotnet/roslyn-project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Input/CommandGroup.cs#L12